core-web/apps/dotcms-ui/src/app/view/components/dot-dialog/dot-dialog.component.ts

```diff
@@ -172,17 +172,10 @@ export class DotDialogComponent implements OnChanges {
     }
 
     private handleKeyboardEvents(event: KeyboardEvent): void {
-        switch (event.code) {
-            case 'Escape':
-                this.cancelAction();
-                break;
-
-            case 'Enter':
-                this.acceptAction();
-                break;
-
-            default:
-                break;
+        if (event.code === 'Escape') {
+            this.cancelAction();
+        } else if (event.code === 'Enter' && (event.metaKey || event.altKey)) {
```

Comment:
Why are the `$event.metaKey` and `$event.altKey` needed?

Reply:
metaKey is for Mac OS 

altKey is for Windows/Linux

Reply:
Cool!